clients/fossid-webapp/src/main/kotlin/FossIdRestService.kt

```diff
@@ -43,18 +44,19 @@ import retrofit2.http.POST
 
 interface FossIdRestService {
     companion object {
+        val FOSSID_JSON_MAPPER = JsonMapper()
```

Comment:
As you anyway need to do another fixup: Please rename this to just `JSON_MAPPER`, as it's already inside `FossIdRestService`.